dashboard/models.py

```diff
@@ -380,14 +378,13 @@ def get_user_in_course_id(self, user, course_id):
         )
 
 class User(models.Model):
-    ENROLLMENT_TYPES = Choices(
-        ('StudentEnrollment', 'Student'),
-        #('StudentViewEnrollment', 'Student View'),
-        ('TaEnrollment', 'Teaching Assistant'),
-        ('TeacherEnrollment', 'Instructor'),
-        #('DesignerEnrollment', 'Designer'),
-        #('ObserverEnrollment', 'Observer'),
-    )
+    class EnrollmentType(models.TextChoices):
```

Comment:
This looks like a good change. I think we'd want to convert this over to using the roles.py in pylti but that can be on a separate issue. 

https://github.com/dmitry-viskov/pylti1.3/blob/master/pylti1p3/roles.py

Reply:
I'm not sure I understand why we'd want to do this. The role data is coming from Unizin/Canvas here, and I don't think it will look the exactly the same. Also, if it did work, wouldn't this result in some very tight coupling? I think choices CVs are included in migrations.

Reply:
Yeah, maybe I was thinking of how we could just use the role from the launch instead of the one in Unizin. However since we allow them to switch courses in the app this probably wouldn't be easily possible. These should semi-closely match LTI but yeah, they're not going to be exactly the same.